features/search/filters.feature

```diff
@@ -132,3 +132,18 @@ Feature: Filters
     Then I should see "A Hobbit's Meandering"
       And I should not see "Bilbo Does the Thing"
       And I should not see "Roonal Woozlib and the Ferrets of Nimh"
+
+  @javascript
+  Scenario: The filter counts should match the actual returned count
+    Given I am logged in as "meatloaf"
+      And I bookmark the work "Bilbo Does the Thing"
+      And I bookmark the work "A Hobbit's Meandering"
+      And I am logged out
+      And I am logged in as "anothermeatloaf"
+      And I bookmark the work "Bilbo Does the Thing"
+      And I bookmark the work "A Hobbit's Meandering"
+      And all indexing jobs have been run
+    When I view the tag "The Hobbit" and go to its bookmarks page
```

Comment:
Would `When I go to the bookmarks tagged "The Hobbit"` work here? It's defined in features/support/paths.rb:
```
    when /^the bookmarks tagged "(.*)"$/i
      step %{all indexing jobs have been run}
      tag_bookmarks_path(Tag.find_by_name($1))
```

Reply:
Got it. It seemed like that step should have existed already, I just struggled to find it. Thanks for pointing this out.